Package.swift

```diff
@@ -30,5 +30,4 @@ let package = Package(
             ],
             path: "TinkMoneyManagerUITarget"
         ),
-    ]
```

Comment:
Looks like this line was accidentally removed.

Reply:
Hmm really strange... I will take a look 👀